### PR TITLE
Add Map to SBI view

### PR DESCRIPTION
**Pull request by FayToward**
https://eaflood.atlassian.net/browse/SFI-332

### CODE REVIEW

**Review by fareed-mohammed — APPROVED** [review]
LGTM

File: app/views/land-business-details/land-business-details.njk [hunk 1]
```diff
@@ -32,11 +32,14 @@
         Bank Top Farm, Warren Lane, Otley LS21 1HT</p>
     <h2 class="govuk-heading-m">Your land</h2>
     <p class="govuk-body">270 hectares of land<br>5100 metres of boundaries</p>
+    <p class="govuk-body"><a href="/map?sbi={{sbi}}">View your land parcels</a></p>
```
Comment (fareed-mohammed) [inline app/views/land-business-details/land-business-details.njk:35]: For the error in the map page, can we find a way to handle it as we have data architects using the sandpit web service and ideally we shouldn't have unhandled exceptions.  Just a simple message saying we hold no data for that SBI will do

**Review by fareed-mohammed — DISMISSED** [review]
For the error in the map page, can we find a way to handle it as we have data architects using the sandpit web service and ideally we shouldn't have unhandled exceptions.  Just a simple message saying we hold no data for that SBI will do

**Review by johnwatson484 — APPROVED** [review]

**Review by fareed-mohammed — APPROVED** [review]
LGTM

**Review by johnwatson484 — APPROVED** [review]
LGTM

**Pull request merged**